core/src/main/java/org/apache/calcite/runtime/SqlFunctions.java

```diff
@@ -5471,6 +5471,10 @@ public static String arrayToString(List list, String delimiter) {
 
   /** SQL {@code ARRAY_TO_STRING(array, delimiter, nullText)} function. */
   public static String arrayToString(List list, String delimiter, @Nullable String nullText) {
+    // Note that the SQL function ARRAY_TO_STRING that we implement will return
+    // 'NULL' when the nullText argument is NULL.  However, that is handled by
+    // the nullPolicy of the RexToLixTranslator.  So here a NULL value
```

Comment:
NIT: redundant spaces (and in above line too)

Reply:
I have changed the summary, removed the spaces, and squashed the commits.